explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+# WoT Thing Description (TD) 1.1 Explainer (DRAFT - WIP)
+This is an updated version of the [Explainer for TD 1.0](Explainer.md).
+
+
+## What's All This About?
+
+[//]: # (What role does the TD specification play in the WoT?)
+
+[//]: # (A brief, 4-5 paragraph explanation of the feature’s value. Outline what the feature does and how it accomplishes those goals in prose. If your feature creates UI, this is a great place to show mocks and user flows.)
+
+[//]: # (The W3C Web of Things [WoT] is intended to enable
+interoperability across IoT Platforms and application domains using Web technology.)
+
+In the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), 
+a Thing is defined as an abstraction of a physical IoT device such as a sensor (temperature, CO2, ...), an actuator (lamp, motor, ...), or a service like a weather service. The Thing Description (TD) provides descriptive metadata for a Thing's interface.
+
+
+With a TD, clients are informed of the choices they can make when interacting with Things such reading a temperature value or switching on a lamp. This can drive applications in the same way that Web HTML pages allow users to
+make choices when navigating the Web and drive Web applications
+through links and forms. Therefore, the TD can be also seen as the ‘index.html’ for Things. The TD itself is serialized in JSON-LD 1.1. 
+
+TDs are designed to provide all the information needed to interact with a Thing. Specifically, it provides answers to questions like those shown in the following figure.
+
+
+![How to Interact with IoT Devices](images/explainer_td.png)
+
+
+## Getting started
+
+Lets assume there is a smart lamp that offers an interface with following characteristics:
+
+| |  ||  
+--- | --- |  --- |
+|Name|MyLampThing| 
+|Protocol |HTTPS| 
+|Security |Basic authentication| 
+|Local server address |192.168.2.134| 
+|Content type |application/json| 
+|Data / function offerings: |status (type: string) | Resource (GET): /status
+|  |toggle | Resource (POST): /toggle
+|  |overheating (type: string) | Resource (GET & longpoll): /oh
+
+
+Based on such information a Thing Description can be simple designed in the following way
```

Comment:
"simply" (or leave it out, "simple" is relative)

Reply:
done